app/views/contexts/calculated_summary_context.py

```diff
@@ -24,7 +25,9 @@ class CalculatedSummaryContext(Context):
     def build_groups_for_section(
         self, section: Mapping, return_to_block_id: str, current_location: Location
     ) -> List[Group]:
-        routing_path = self._router.routing_path(section["id"])
+        routing_path = self._router.routing_path(
+            section["id"], list_item_id=current_location.list_item_id
+        )
```

Comment:
How did repeats work without this?. What is this fixing and is there a test?.

Reply:
Removed it now as per the comment below 👍